store/user.ts

```diff
@@ -19,12 +21,16 @@ const userSlice = createSlice({
       state = action.payload
       return state
     },
+    setUserProfile: (state, action: PayloadAction<UserProfileType>) => {
+      state = { ...initialState, ...action.payload }
```

Comment:
```suggestion
      state = { ...state, ...action.payload }
```

initialState로 하면 모든 값이 초기화 되고 값이 할당 될거야
전 값을 유지하고 싶으면 state로 하는 게 맞을 것 같은데
혹시 initialState로 한 이유가 있을까?

Reply:
아니요 gauth만들고 오류 찾고 이것저것 하면서 하다보니깐 state존재를 잊고 있었음 수정할게